app/components/AssetTree/AssetNode/AssetNode.js

```diff
@@ -26,13 +26,13 @@ const StyledTreeNode = styled.div`
   flex-direction: row;
   align-items: center;
   padding: 5px 8px;
-  padding-left: ${props => getPaddingLeft(props.level)}px;
+  padding-left: ${props => getPaddingLeft(props.$level)}px;
   ${props => (props.selected ? 'background-color: #eee;' : null)}
 `;
 
 const NodeIcon = styled.div`
   font-size: 12px;
-  margin-right: ${props => (props.marginRight ? props.marginRight : 5)}px;
+  margin-right: ${props => (props.$marginright ? props.$marginright : 5)}px;
 `;
 
 const StyledInput = styled.input`
```

Comment:
changes in file `app/components/AssetTree/AssetNode/AssetNode.js` can be reverted as we are not using `style-components` version 6

Reply:
No, actually these transient props thing came with v5 only. That is why I kept these changes.

Reply:
Okay, thank you for the information. We can keep it then, I have yet to test it with version 5, but I believe it might not be causing any errors as you might have already tested these changes.